controller/array_action/array_mediator_svc.py

```diff
@@ -367,19 +367,10 @@ def unmap_volume(self, volume_id, host_name):
         except Exception as ex:
             logger.exception(ex)
             raise ex
-
-
+    
     def get_array_iscsi_name(self):
         logger.debug("Getting array nodes iscsi name")
         nodes_list = self.client.svcinfo.lsnode().as_list
```

Comment:
you don't need .as_list, all the commands return an iterator, you can use it in for...in... directly

Reply:
ok, I will update it